src/main.cpp

```diff
@@ -77,6 +85,9 @@ int main(int argc, char *argv[])
 
                 auto r = cam.get_ray(u, v);
                 col += ray_color(r, world, 0);
+
+                progress += increment;
+                std::cout << "Progress: " << std::setw(5) << progress * 100.0f << "%" << std::endl;
```

Comment:
I have _no idea_ why the `std::setw(5)` is not working, right now.
If you have time, please test it.